.github/workflows/schedule_tests.yml

```
@@ -37,6 +37,37 @@ jobs:
       - name: Run tests
         run: python tests/runtests.py -v2
 
+  pyc-only:
+    runs-on: ubuntu-latest
+    name: .pyc-only
```

Could you use a name that is not preceded by "."? I know is valid but I can image a dark corner somewhere where this is not listed because it starts with a dot.
```suggestion
    name: Python with no source files (pyc only files)
```